src/Stratis.Features.SQLiteWalletRepository/Tables/DBTable.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Globalization;
+using NBitcoin;
+
+namespace Stratis.Features.SQLiteWalletRepository.Tables
+{
+    internal class DBTable
+    {
+        // Intended to handle string, int, long and decimal.
+        internal static string DBParameter(object prop)
```

Comment:
Could this comment be extended to describe the problem being solved here?

Reply:
Done.